### ISSUE TITLE
Fine Tune with Japanese model but generated unstandable audio (Already converted to hiragana)

### ISSUE THREAD

**Issue by Chloey-TS** [SWivid/F5-TTS#1167]
### Checks

- [x] This template is only for usage issues encountered.
- [x] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [x] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [x] I am using English to submit this issue to facilitate community communication.

### Environment Details

System: Linux 6.12.34-1-MANJARO
Python: 3.10.18
Torch: 2.9.0.dev20250720+cu129
Gradio: 5.38.1
GPU: 12GB RTX5080

### Steps to Reproduce

I downloaded Japanese model from [here](https://huggingface.co/Jmica/F5TTS/tree/main/JA_21999120) , and follow [this guide #143](https://github.com/SWivid/F5-TTS/discussions/143) to fine tune in gradio UI interface.

I skipped Transcribe Data and Vocab Check, because I already did it before, I already have wav folder and metadata.csv file in my project folder, I also checked issues converted all kanji to Hiragana.

In prepare data tab, I click prepare, the generated vocab.txt just 11.3 kb, so I replaced it with downloaded Japanese model's vocab text, which is 15.9kb, then I go to train model tab, here is my settings and output:

<img width="2431" height="1355" alt="Image" src="https://github.com/user-attachments/assets/c98d57b1-eae9-4f11-bdde-78ed30c522ad" />
 
```
Starting app...
* Running on local URL:  http://127.0.0.1:7860
* To create a public link, set `share=True` in `launch()`.
run command : 
accelerate launch --mixed_precision=fp16 "/home/ai/Projects/AI_Voice/F5-TTS/src/f5_tts/train/finetune_cli.py" --exp_name F5TTS_v1_Base --learning_rate 1e-05 --batch_size_per_gpu 1600 --batch_size_type frame --max_samples 12 --grad_accumulation_steps 1 --max_grad_norm 1 --epochs 51 --num_warmup_updates 50 --save_per_updates 100 --keep_last_n_checkpoints 1 --last_per_updates 100 --dataset_name fuku --finetune --pretrain "/home/ai/Projects/AI_Voice/F5-TTS/ckpts/Japanese/model_21999120.pt" --tokenizer pinyin --log_samples

ipex flag is deprecated, will be removed in Accelerate v1.10. From 2.7.0, PyTorch has all needed optimizations for Intel CPU and XPU.
copy checkpoint for finetune

vocab :  2976

vocoder :  vocos
Using logger: None
Loading dataset ...
Download Vocos from huggingface charactr/vocos-mel-24khz

Sorting with sampler... if slow, check whether dataset is provided with duration:   0%|          | 0/51 [00:00<?, ?it/s]
Sorting with sampler... if slow, check whether dataset is provided with duration: 100%|██████████| 51/51 [00:00<00:00, 838860.80it/s]

Creating dynamic batches with 1600 audio frames per gpu:   0%|          | 0/51 [00:00<?, ?it/s]
Creating dynamic batches with 1600 audio frames per gpu: 100%|██████████| 51/51 [00:00<00:00, 1465133.59it/s]
Saved last checkpoint at update 21999120

Download Vocos from huggingface charactr/vocos-mel-24khz

vocab :  /home/ai/Projects/AI_Voice/F5-TTS/src/f5_tts/../../data/fuku_pinyin/vocab.txt
token :  custom
model :  /home/ai/Projects/AI_Voice/F5-TTS/src/f5_tts/../../ckpts/fuku/pretrained_model_21999120.pt 

update >>  None /home/ai/Projects/AI_Voice/F5-TTS/src/f5_tts/../../ckpts/fuku/pretrained_model_21999120.pt True
Converting audio...
Using custom reference text...

ref_text   たいていはあたまにごむせいのずきんをかぶって、えびちゃやこんやあいのいろをなみまにうかしていた。そういうありさまをもくげきしたばかりのわたくしのめには、. 
/home/ai/miniconda3/envs/f5-tts/lib/python3.10/site-packages/torchaudio/_backend/utils.py:213: UserWarning: In 2.9, this function's implementation will be changed to use torchaudio.load_with_torchcodec` under the hood. Some parameters like ``normalize``, ``format``, ``buffer_size``, and ``backend`` will be ignored. We recommend that you port your code to rely directly on TorchCodec's decoder instead: https://docs.pytorch.org/torchcodec/stable/generated/torchcodec.decoders.AudioDecoder.html#torchcodec.decoders.AudioDecoder.
  warnings.warn(
gen_text 0 たいていはあたまにごむせいのずきんをかぶって、えびちゃやこんやあいのいろをなみまにうかしていた。そういうありさまをもくげきしたばかりのわたくしのめには、


Generating audio in 1 batches...
100%|██████████████████████████████████████████████████████████████████████| 1/1 [00:03<00:00,  3.01s/it]
Converting audio...
Using custom reference text...

ref_text   これはせけんをはばかるえんりょというよりも、そのほうがわたくしにとってしぜんだからである。わたくしはそのひとのきおくをよびおこすごとに、すぐせんせいといいたくなる。
gen_text 0 これはせけんをはばかるえんりょというよりも、そのほうがわたくしにとってしぜんだからである。
gen_text 1 わたくしはそのひとのきおくをよびおこすごとに、すぐせんせいといいたくなる。
```

and here is screenshot of test model tab:

<img width="2455" height="1415" alt="Image" src="https://github.com/user-attachments/assets/d4c05f5c-bf28-4b61-a217-fa0bb11d15a3" />

I've been `accelerate config` before, no matter I choose same none precious or fp16, all generated unstanedable audio.
btw, I tried this Japanese model for infer, which works fine.

Thanks for reading, and any help is appreciated.

### ✔️ Expected Behavior

I hope it can generated normal Japanese audio after fine tune

### ❌ Actual Behavior

generated unstandable audio, neither Japanese or English or Chinese.

**Comment by SWivid** [comment]
try prune the checkpoint from https://huggingface.co/Jmica/F5TTS/tree/main/JA_21999120 first, which is to leave only ema or non-ema model weight and remove the optimzer scheduler and other state dicts.

finetune_gradio ui interface has a pruning function but not sure if compatible.
basically you could manually do that: torch load the .pt file -> you get the dict -> remove the unwanted stuff -> save to a .pt file

**Comment by Chloey-TS** [comment]
Hi @SWivid 
Thanks for your reply.
This method works, I use torch load the pt file and only save the ema_model_state_dict and save pt file again, use this pt file to finetune, the generated Japanese audio is good.
Thanks for your help, you really saved my day.
Have a great day!
